sycl/include/sycl/platform.hpp

```diff
@@ -144,9 +183,17 @@ class __SYCL_EXPORT platform : public detail::OwnerLessBase<platform> {
   /// Queries this SYCL platform for info.
   ///
   /// The return type depends on information being queried.
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+  template <typename Param>
+  typename detail::is_platform_info_desc<Param>::return_type get_info() const {
+    auto Info = get_info_impl<Param>();
+    return convert_from_abi_neutral<typename detail::ABINeutralT_t<
```

Comment:
You don't need to specify the template type parameter here - it should be deduced from the operand.

Reply:
I don't see how my request contradicts that.

Reply:
Sure, I will go ahead and change to a normal function.

Reply:
That's not what I asked. You can have
```
template <typename T>
void foo(T x) {...}
```

yet still call it as

```
std::string x;
foo(x);
```

and not as

```
std::string x
foo<std::string>(x);
```

Reply:
I really appreciate your elaboration of details.
You are absolutely correct.
I will change it accordingly.
Thanks.